meta-genivi-dev/meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -1,175 +1,10 @@
 # Recipe created by recipetool
-# This is the basis of a recipe and may need further editing in order to be fully functional.
-# (Feel free to remove these comments when editing.)
 
 SUMMARY = "zebkit"
 HOMEPAGE = "https://github.com/barmalei/zebra"
-# WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
-# your responsibility to verify that the values are complete and correct.
```

Comment:
@rzr 
> "why remove ?"

Because I asked for it.

I don't want recipes that say "these are best guesses".   The recipe author should take the responsibility to provide correct license information.  The comments are there because a human author is assumed to be able to do that more carefully and more correctly than an automated script.  The final result should not say we don't know if this is correct.    

The reason that text is there is because the recipe has been auto-generated from a script.  And the script is of course possibly making a mistake, "best guess".

..."the following LICENSE and LIC_FILES_CHKSUM values are best guesses it is your responsibility to verify that the values are complete and correct."

The script is instructing _the recipe author_ to check and verify that the license information is correct before considering the recipe to be finished and ready to use.

> "if you pull code you need to comply licenses"

Everyone knows that and that has nothing to do with removing these comments.  In fact, to help adopters comply with licenses, we should not have comments that suggest that the license information is in any way unclear.

I think maybe there was just some language misunderstanding here?

@akilawickey I'm sorry you get contradicting instructions from multiple people.